EAP adapter tests not working

### Before reporting an issue

- [x] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

ci

### Describe the bug

```
2025-04-29 09:57:37  -- [ERROR] org.keycloak.testsuite.adapter.servlet.SAMLClockSkewAdapterTest.testTokenIsNotYetValid  Time elapsed: 1.988 s  <<< ERROR!
2025-04-29 09:57:37  -- java.lang.IllegalStateException: The server should be WildFly (either ManagementProtocol.HTTP(S)_REMOTING was used or the 'creaper.wildfly' system property was set), but client libraries are AS7-only
2025-04-29 09:57:37  -- 	at org.wildfly.extras.creaper.core.online.OnlineOptions.createModelControllerClient(OnlineOptions.java:448)
2025-04-29 09:57:37  -- 	at org.wildfly.extras.creaper.core.online.OnlineManagementClientImpl.connect(OnlineManagementClientImpl.java:42)
2025-04-29 09:57:37  -- 	at org.wildfly.extras.creaper.core.online.OnlineManagementClientImpl.<init>(OnlineManagementClientImpl.java:38)
2025-04-29 09:57:37  -- 	at org.wildfly.extras.creaper.core.online.OnlineOptions$2.create(OnlineOptions.java:534)
2025-04-29 09:57:37  -- 	at org.wildfly.extras.creaper.core.ManagementClient.online(ManagementClient.java:34)
2025-04-29 09:57:37  -- 	at org.keycloak.testsuite.arquillian.AppServerTestEnricher.getManagementClient(AppServerTestEnricher.java:184)
2025-04-29 09:57:37  -- 	at org.keycloak.testsuite.arquillian.AppServerTestEnricher.getManagementClient(AppServerTestEnricher.java:179)
2025-04-29 09:57:37  -- 	at org.keycloak.testsuite.arquillian.AppServerTestEnricher.enableHTTPSForAppServer(AppServerTestEnricher.java:253)
2025-04-29 09:57:37  -- 	at org.keycloak.testsuite.adapter.AbstractAdapterTest.setUpAppServer(AbstractAdapterTest.java:73)
```

https://keycloak-jenkins/view/Keycloak/job/keycloak-continuous-testing/249/execution/node/334/log/

### Version

26.0.11

### Regression

- [x] The issue is a regression

### Expected behavior

-

### Actual behavior

-

### How to Reproduce?

-

### Anything else?

Wildfly adapter tests are passing upstream, but for EAP they are failing.

--- a/testsuite/integration-arquillian/pom.xml
+++ b/testsuite/integration-arquillian/pom.xml
@@ -54,7 +54,7 @@
         <arquillian-infinispan-container.version>1.2.0.Beta3</arquillian-infinispan-container.version>
         <undertow.version>${undertow-jakarta.version}</undertow.version>
         <undertow-embedded.version>1.0.0.Final</undertow-embedded.version>
-        <version.org.wildfly.extras.creaper>1.6.1</version.org.wildfly.extras.creaper>
+        <version.org.wildfly.extras.creaper>2.0.3</version.org.wildfly.extras.creaper>
         <jakarta.persistence-legacy.version>2.2.3</jakarta.persistence-legacy.version>
         <smallrye.jandex.version>3.0.5</smallrye.jandex.version>
         <commons.validator.version>1.8.0</commons.validator.version>

--- a/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/AppServerTestEnricher.java
+++ b/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/AppServerTestEnricher.java
@@ -214,37 +214,21 @@ public class AppServerTestEnricher {
         Administration administration = new Administration(client);
         Operations operations = new Operations(client);
 
-        boolean isElytronConfigured = false;
-        try {
-            // check if the eap is configured to use elytron
-            ModelNodeResult result = operations.readAttribute(Address.subsystem("undertow")
-                    .and("server", "default-server").and("https-listener", "https"), "ssl-context");
-            if (!result.isFailed() && result.hasDefinedValue()) {
-                isElytronConfigured = true;
-            }
-        } catch (IOException e) {
-            log.debug("Error reading 'ssl-context' attribute in undertow, assuming not elytron", e);
-        }
-
-        if (isElytronConfigured && !operations.exists(Address.subsystem("elytron").and("server-ssl-context", "KCSslContext"))) {
+        if (!operations.exists(Address.subsystem("elytron").and("server-ssl-context", "KCSslContext"))) {
             client.execute("/subsystem=elytron/key-store=KCKeyStore:add(path=adapter.jks,relative-to=jboss.server.config.dir,credential-reference={clear-text=secret},type=JKS)");
             client.execute("/subsystem=elytron/key-manager=KCKeyManager:add(key-store=KCKeyStore,credential-reference={clear-text=secret,algorithm=PKIX})");
             client.execute("/subsystem=elytron/key-store=KCTrustStore:add(relative-to=jboss.server.config.dir,path=keycloak.truststore,credential-reference={clear-text=secret},type=JKS)");
             client.execute("/subsystem=elytron/trust-manager=KCTrustManager:add(key-store=KCTrustStore)");
             client.execute("/subsystem=elytron/server-ssl-context=KCSslContext:add(key-manager=KCKeyManager,trust-manager=KCTrustManager)");
-            client.execute("/subsystem=undertow/server=default-server/https-listener=https:write-attribute(name=ssl-context,value=KCSslContext)");
-        } else if (!operations.exists(Address.coreService("management").and("security-realm", "UndertowRealm"))) {
-            client.execute("/core-service=management/security-realm=UndertowRealm:add()");
-            client.execute("/core-service=management/security-realm=UndertowRealm/server-identity=ssl:add(keystore-relative-to=jboss.server.config.dir,keystore-password=secret,keystore-path=adapter.jks");
+            if (operations.exists(Address.subsystem("undertow").and("server", "default-server").and("https-listener", "https"))) {
+                client.execute("/subsystem=undertow/server=default-server/https-listener=https:write-attribute(name=ssl-context,value=KCSslContext)");
+            } else {
+                client.execute("/subsystem=undertow/server=default-server/https-listener=https:add(socket-binding=https, enable-http2=true, ssl-context=KCSslContext)");
+            }
         }
 
         client.execute("/system-property=javax.net.ssl.trustStore:add(value=${jboss.server.config.dir}/keycloak.truststore)");
         client.execute("/system-property=javax.net.ssl.trustStorePassword:add(value=secret)");
-
-        if (!isElytronConfigured) {
-            removeHttpsListener(client, administration);
-            addHttpsListenerAppServer(client);
-        }
 
         reloadOrRestartTimeoutClient(administration);
     }

